app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java

```diff
@@ -176,7 +176,7 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
      */
     private void createAlbum() {
         Context c = this.getApplicationContext();
-        PXLClient.initialize("196i8ZzIAhKU8dO2kDe");
+        PXLClient.initialize("196i8ZzIAhKU8dO2kDe", "66f3c350bc162c143eecd41711");
```

Comment:
Should remove the secret_key and cleanup the commits with them.

Reply:
oh my god lol bfg this please this is a public repo